### PR TITLE
Add settings for drtelx and redbox

### PR DESCRIPTION
**Pull request by samkotlove**
Includes some consolidation of pharmacy ids as well

### CODE REVIEW

File: packages/settings/src/pharmacies.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const ALTO_ID = 'phr_01G9CM93X1NFP1C9H9K50DPKHX';
```
Comment (captDaylight) [inline packages/settings/src/pharmacies.ts]: should we make this `pharmacies.photon.ts`?

File: packages/settings/src/pharmacies.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const ALTO_PHARMACY_ID = 'phr_01G9CM93X1NFP1C9H9K50DPKHX';
```
Comment (jknapp25) [inline packages/settings/src/pharmacies.ts:1]: these are same across envs?

Comment (captDaylight) [reply packages/settings/src/pharmacies.ts:1]: ha writing the same thing, maybe make it `pharmacies.photon.ts`

**Comment by jknapp25** [comment]
need branding for drtelx?

File: packages/settings/src/pharmacies.ts (hunk 2 above)
Comment (captDaylight) [reply packages/settings/src/pharmacies.ts:1]: yup!

**Review by captDaylight — APPROVED** [review]

File: packages/settings/src/pharmacies.ts (hunk 1 above)
Comment (samkotlove) [reply packages/settings/src/pharmacies.ts]: The IDs are the same in all environments

**Pull request merged**